config/application-OS-QA.properties

```diff
@@ -12,12 +12,12 @@ repeat.maxRuns=2400
 
 ## Set to the QA SPE course to use.  Both test and production sites are in production Canvas.
 ## production SPE canvas course
-#getgrades.COURSEID=159923
+getgrades.COURSEID=159923
 ## test course
-getgrades.COURSEID=187539
+#getgrades.COURSEID=187539
 
-email.mail.host=mail-relay.itd.umich.edu
-email.mail.smtp.port=25
+#email.mail.host=mail-relay.itd.umich.edu
```

Comment:
With the change, are we sitll sending emails?

Reply:
You can't tell from a single file if the property setting will be overridden elsewhere.  Commenting this out means the setting in this file won't be used.  

Spring Boot provides a very useful but slightly complex set of ways to set properties.  SPE uses Spring profiles and environment variables to allow setting and overriding property values.